fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1133,6 +1133,10 @@ public void loadImage(String imageDir) throws IOException, DdlException {
             checksum = analyzeManager.loadAnalyze(dis, checksum);
             remoteChecksum = dis.readLong();
             checksum = workGroupMgr.loadWorkGroups(dis, checksum);
+            if (dis.available() > 0) {
```

Comment:
availabe() <=0 dose not means the end of file. It just means that there is no more data in the current buffer.

Reply:
You can just remove this check, because there is catch of EOFException in process of read image.

Reply:
done